### PR TITLE
Neural Web directory added

### PR DESCRIPTION
**Pull request by Korinin38**
<code>github = 'musorka'</code>

### CODE REVIEW

File: Neural Web/gitTest/rtr.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+rtr
```
Comment (evsey9) [inline Neural Web/gitTest/rtr.txt:1]: Please follow proper formatting rules and make sure there is a () and ;

**Review by evsey9 — CHANGES_REQUESTED** [review]
Please do what i asked in the code

File: Neural Web/gitTest/rtr.txt (hunk 1 above)
Comment (Korinin38) [reply Neural Web/gitTest/rtr.txt:1]: ok
<code>();</code>

Comment (evsey9) [reply Neural Web/gitTest/rtr.txt:1]: блин ты победил((

**Review by evsey9 — APPROVED** [review]

**Pull request merged**